axelrod/match.py

```diff
@@ -100,72 +96,38 @@ def play(self):
 
     def scores(self, game=None):
         """Returns the scores of the previous Match plays."""
-        if not game:
-            game = Game()
-        scores = [game.score(plays) for plays in self.result]
+        scores = iu.compute_scores(self.result, game)
         return scores
 
```

Comment:
Why bother with the ```scores``` variable instead of simply returning the result of ```iu.compute_scores``` in a single line?